src/guides/v2.4/graphql/interfaces/customizable-option-interface.md

```diff
@@ -268,3 +268,75 @@ The following query returns information about the customizable options configure
   }
 }
 ```
+
+The following query returns information about the customizable options configured for the product with a `sku` of `xyz` with Custom Option type Text Field.
+
+- Custom option Option Type is text field with required field.
```

Comment:
I saw that you were still getting Linting errors and I checked the rest of the file. The problem is that there is a list earlier in the file with style -
*  Text area
*  Checkbox
*  Date picker

You can only have one list style per file, so you need to change your list to use asterisks